opm/simulators/aquifers/BlackoilAquiferModel_impl.hpp

```diff
@@ -1,225 +1,208 @@
 #include <opm/grid/utility/cartesianToCompressed.hpp>
-namespace Opm {
+namespace Opm
+{
 
-  template<typename TypeTag>
-  BlackoilAquiferModel<TypeTag>::
-  BlackoilAquiferModel(Simulator& simulator)
-  : simulator_(simulator)
-  {
+template <typename TypeTag>
+BlackoilAquiferModel<TypeTag>::BlackoilAquiferModel(Simulator& simulator)
+    : simulator_(simulator)
+{
     init();
-  }
-
-  template<typename TypeTag>
-  void
-  BlackoilAquiferModel<TypeTag>::initialSolutionApplied()
-  {
-    if(aquiferCarterTracyActive())
-    {
-      for (auto aquifer = aquifers_CarterTracy.begin(); aquifer != aquifers_CarterTracy.end(); ++aquifer)
-      {
-        aquifer->initialSolutionApplied();
-      }
+}
+
+template <typename TypeTag>
+void
+BlackoilAquiferModel<TypeTag>::initialSolutionApplied()
+{
+    if (aquiferCarterTracyActive()) {
+        for (auto aquifer = aquifers_CarterTracy.begin(); aquifer != aquifers_CarterTracy.end(); ++aquifer) {
+            aquifer->initialSolutionApplied();
+        }
```

Comment:
While you're already here, would you mind switching this (and the Fetkovich branch too) into using range-`for`?

Reply:
Sure.  Should we remove the usage of `at`?